browser/app/js/buckets/BucketList.js

```diff
@@ -42,32 +42,26 @@ export class BucketList extends React.Component {
   render() {
     const { visibleBuckets } = this.props
     return (
-      <div className="fesl-inner">
-        <Scrollbars
-          renderTrackVertical={props => <div className="scrollbar-vertical" />}
-        >
-          <ul>
-            {visibleBuckets.map(bucket => (
-              <BucketContainer key={bucket} bucket={bucket} />
-            ))}
-          </ul>
-        </Scrollbars>
+      <div className="buckets__list">
```

Comment:
Because the Scrollbars component is removed here, we aren't able to scroll on the list of buckets when we have many buckets. We should add it back and apply the relevant CSS changes.